include/rocksdb/write_batch.h

```diff
@@ -343,55 +360,26 @@ class WriteBatch : public WriteBatchBase {
   bool HasRollback() const;
 
   // Experimental.
-  // Assign timestamp to write batch.
+  //
+  // Update timestamps of existing entries in the write batch if
+  // applicable. If a key is intended for a column family that disables
+  // timestamp, then this API won't set the timestamp for this key.
```

Comment:
Thanks for the polish / updates. Am I correct that `checker` is used to determine whether a "key is intended for a column family that disables timestamp"? I do not like the name `checker` for something that determines what operation to perform or how. To me the name is only appropriate for something that injects an "accept"/"reject" decision. You might say that the checker rejects updating timestamps where timestamp is disabled, but to me that is determining what operation to perform, because in that case "no update" is success.

Reply:
SGTM. How about renaming 'checker' to 'get_timestamp_size_func'?